dace/data.py

```diff
@@ -354,6 +354,163 @@ def add(X: dace.float32[10, 10] @ dace.StorageType.GPU_Global):
         return new_desc
 
 
+def _arrays_to_json(arrays):
+    if arrays is None:
+        return None
+    return [(k, serialize.to_json(v)) for k, v in arrays.items()]
+
+
+def _arrays_from_json(obj, context=None):
+    if obj is None:
+        return {}
+    return OrderedDict((k, serialize.from_json(v, context)) for k, v in obj)
+
+
+@make_properties
+class Structure(Data):
+    """ Base class for structures. """
+
+    members = Property(dtype=OrderedDict,
```

Comment:
For the VSCode plugin we probably want to still make it OrderedDictProperty with the methods. @phschaad is that true?

Reply:
That would be better, yes

Reply:
OK, but note that this functionality is not intended and I need to change the Property class. The thing is that `from/to_json` is a (Python) property in the base class. However, OrderedDictProperty replaces these properties with bound methods. `Property.__init__` sets `_from/to_json` (note the underscore at the beginning), which does nothing if the corresponding (Python) property does not exist (and it doesn't exist because it was completely replaced by a bound method ...). I can fix this in `Property.__init__` by checking for this and (instead/also) replacing the names without the underscore. It looks bad though.